pom.xml

```diff
@@ -804,6 +805,43 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-enforcer-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>${maven-checkstyle-plugin.version}</version>
+				<configuration>
+					<checkstyleRules>
+						<module name = "Checker">
+							<property name="fileExtensions" value="java,js,css,xml"/>
```

Comment:
@sebawagner `fileExtensions` were added - but it doesn't work ...

Reply:
I think you enabled the file extensions successfully. 

But TreeWalker only works for Java file types. https://checkstyle.sourceforge.io/config.html#TreeWalker
> FileSetCheck TreeWalker checks individual **Java source** files and defines properties that are applicable to checking such files.

I think js you can forget, but the google reference checkstyle config checks additionally XML and properties files.

https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml

And those rules outside of TreeWalker in above google XML should also work for non Java files. Except they have additional rules.

But yeah its really Java centric. For JS I think you should have a linter filer and invoke a eslint. You can just run Eslint from command line. Or invoke it via a maven wrapper that invokes a command line. 
And it will be much better (and more easy) to re-use eslint files from other projects for JavaScript checks. And prob also more what people expect that code Javascript.